PeakLocalisations/Localisation/Core/Stats.xcstrings

```diff
@@ -170,6 +224,12 @@
             "state" : "translated",
             "value" : "Number of Workouts"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Número de Entrenos"
```

Comment:
same as above, Cantidad is always preferable for number of countable things

Reply:
Is that for Spain, Latin America, or both? Apple seems to use cantidad sparingly, and even then only in Latin America https://applelocalization.com/?q=Number+of+notifications&l=English&l=Spanish

Reply:
I think it's for everywhere. Número for cantidad is not unheard of but IMO a lot less used.

Reply:
I think I'm gonna leave this as número for the Spain translation, just for consistency with Apple, and I'll use cantidad for the Latin American translation